### PR TITLE
Rewrite "Creating a user" as "Managing users"

### PR DESCRIPTION
**Pull request by kjac**
## Description

The ["Creating a user"](https://docs.umbraco.com/umbraco-cms/reference/management/services/create-a-new-user) article references obsolete and removed classes and service methods. It is also not at all about creating users - it is about adding users to groups 😆 

I have rewritten the entire article to get the code samples etc. up to date with V14. I have also opted to rename the article to "Managing Users". Eventually we could add more samples to this - for example, something about creating users? 😛 

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

V14

## Deadline (if relevant)

Whenever possible.

### CODE REVIEW

**Comment by jonat123** [comment]
@kjac Thanks for creating this PR 😄

We will review it as soon as we can 😊

File: 14/umbraco-cms/reference/management/services/managing-users.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+description: This will show you how to perform various User management using the Umbraco service layer.
+---
+
+# Managing Users
+
+## Assigning a User to a User Group
+
+To assign a User to a User Group, we need both the `IUserService` and `IUserGroupService`. As with all Umbraco services, these are obtained using dependency injection.
+
+1. We start by defining an interface for our implementation:
```
Comment (jonat123) [inline 14/umbraco-cms/reference/management/services/managing-users.md:11]:
```suggestion
1. Start by defining an interface for our implementation:
```

File: 14/umbraco-cms/reference/management/services/managing-users.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            performingUserKey
+        );
+
+        return result.Success;
+    }
+}
+```
+{% endcode %}
+
+3. To register the implementation, we need a Composer:
```
Comment (jonat123) [inline 14/umbraco-cms/reference/management/services/managing-users.md:66]:
```suggestion
3. Register the implementation in a Composer:
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@kjac Looks good!

Just have two suggestions 😄

SInce you are removing an article, we also need a redirect in place.

Can you add the following line to the .gitbook.yaml file:
`reference/management/services/create-a-new-user: reference/management/services/managing-users.md`

you can find the file in the 14 -> umbraco-cms folder 😄

**Comment by kjac** [comment]
Done @jonat123 👍

**Review by jonat123 — APPROVED** [review]
@kjac LGTM 💪

Merging it in!

**Pull request merged**